plugins/modules/elb_target_group.py

```diff
@@ -767,6 +780,9 @@ def create_or_update_target_group(connection, module):
     if deregistration_delay_timeout is not None:
         if str(deregistration_delay_timeout) != current_tg_attributes['deregistration_delay_timeout_seconds']:
             update_attributes.append({'Key': 'deregistration_delay.timeout_seconds', 'Value': str(deregistration_delay_timeout)})
+    if deregistration_connection_termination is not None:
+        if deregistration_connection_termination and current_tg_attributes.get('deregistration_delay_connection_termination_enabled') != "true":
```

Comment:
The AWS API seems to be a little inconsistent and needed `""` around the `bool` value like done here:
https://github.com/ansible-collections/community.aws/pull/913/files#diff-2c1c6b6915c4678f611809c13fca7baf3b17b239efd502403f050f52d2f03eafR787